modules/log6x-logging-clo-cli-install.adoc

```diff
@@ -0,0 +1,165 @@
+// Module is included in the following assemblies:
+//
+// 
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-installing-logging-operator_{context}"]
+= Installing {clo} by using the CLI
+
+After you have installed the log store, install {clo}, which collects and forwards the logs to the log store. This can be done from the {product-title} command-line interface (CLI).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+* You installed and configured {loki-op}.
+* You have created the `openshift-logging` namespace.
+
+.Procedure
+
+. Create a `Subscription` object for {clo}:
++
+.Example `Subscription` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable-6.<y> # <2>
+  installPlanApproval: Automatic # <3>
+  name: cluster-logging
+  source: redhat-operators # <4>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable-6.<y>` as the channel.
+<3> If the approval strategy in the subscription is set to `Automatic`, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to `Manual`, you must manually approve pending updates.
+<4> Specify `redhat-operators` as the value. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object that you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object:
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: ClusterLogForwarder.v1.observability.openshift.io,LogFileMetricExporter.v1alpha1.logging.openshift.io
```

Comment:
No need to add the annotations in the yaml file.

Reply:
Not adding the annotation will not break anything on the UI? I asked @theashiot to add the annotation since it showed when installing the operator using the UI

Reply:
No. The annotation is added automatically.

Reply:
removed